caab-api/open-api-specification.yml

```diff
@@ -175,7 +175,72 @@ paths:
             type: 'integer'
             format: 'int64'
             example: '1234567890'
-        - name: 'Caab-user-Login-Id'
+        - name: 'Caab-User-Login-Id'
+          in: header
+          required: true
+          schema:
+            type: 'string'
+          example: 'MARILYN@DESORANDCO.CO.UK'
```

Comment:
Isn't this a real provider login? Maybe shouldn't be used as an example.

Reply:
amended